### ISSUE TITLE
Payload dist output is to heavy

### ISSUE THREAD

**Issue by TarasOP** [payloadcms/payload#5397]
### Describe the Bug

I want to deploy the payload app to the amplify hosting. But the output is greater than 2220 MB, so the error occurs. 

### To Reproduce

You can create a payload project and run the build command. Then tun `du -h -d1 . | sort -h`

### Payload Version

2.11.2

### Adapters and Plugins

@payloadcms/plugin-form-builder, @payloadcms/db-mongodb

**Comment by DanRibbens** [comment]
Yes, in the next version of payload, v3, the dependencies are being reduced drastically. That is a focus we have been working on and will be in beta in a few weeks. 

Usually the dependencies for a Payload project are okay with 1GB of memory. Make sure you don't have any extras in your dependencies that could be moved to devDependencies.

If you post your package.json we can try and point out improvements. 
I'm converting this to a discussion in the meantime.
